pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -254,6 +255,45 @@ var _ = Describe("Linseed rendering tests", func() {
 			Expect(d.Spec.Template.Spec.Containers[0].Env).To(ContainElement(corev1.EnvVar{Name: "LINSEED_FIPS_MODE_ENABLED", Value: "true"}))
 		})
 	})
+	Context("multi-tenant rendering", func() {
+		var installation *operatorv1.InstallationSpec
+		var replicas int32
+		var cfg *Config
+		clusterDomain := "cluster.local"
+		esClusterConfig := relasticsearch.NewClusterConfig("", 1, 1, 1)
+		BeforeEach(func() {
```

Comment:
Same nit here - a newline before `BeforeEach` and `It` clauses as well, just to keep the code a smidge more readable.